File: practica2/c++/progSuma.cc
#include <vector>
#include "progSuma.h"
#include "read.h"
#include "suma.h"
#include "write.h"

progSuma::progSuma()
: programa::programa(4)
{
	instrucciones[0] = new read();
	instrucciones[1] = new read();
	instrucciones[2] = new add();
	instrucciones[3] = new write();
}

File: practica2/java/ProgSuma.java
/**
 * progSuma
 */
public class ProgSuma extends Programa{
	public ProgSuma()
   {
      super(4);
      instrucciones[0] = new Read();
      instrucciones[1] = new Read();
      instrucciones[2] = new Suma();
      instrucciones[3] = new Write();
   }
}

    
